pkg/clusternetwork/config.go

```diff
@@ -84,6 +96,11 @@ func (ovn *ovnKubernetes) GetServiceCIDR() (string, error) {
 	return ovn.clusterNetworkConfig.serviceCIDR, nil
 }
 
+// GetVXLANPort gets the vxlan port to be used for vxlan tunnel establishment
+func (ovn *ovnKubernetes) GetVXLANPort() (string, error) {
```

Comment:
Why is this function returning an error?

Reply:
Ya, it was a miss on our side earlier, actually these getters won't return an error at all. I just followed the pattern. We should change this perhaps in another PR.